package.json

```diff
@@ -33,10 +34,15 @@
     "auth",
     "login"
   ],
+  "dependencies": {
+    "Base64": "0.3.0",
+    "q": "1.4.1",
+    "reqwest": "2.0.5",
+    "xhr2": "0.1.3"
```

Comment:
If you `npm install @okta/okta-auth-js`, it will complain that `reqwest` has a peer dependency of `xhr2`, so I include it to avoid the warning